src/components/ProfileInfos/index.js

```diff
@@ -299,68 +306,76 @@ const ProfileInfos = () => {
                                 <Grid item xs>
                                     <form className={classes.form}>
                                         <div> 
-                                                                                    
+                                            <Checkbox color='primary' className={classes.checkboxField} />                                        
                                             <LabelWrapper> Firstname:</LabelWrapper>
-                                            <InputWrapper>
```

Comment:
You can make this input visible.
<img width="1440" alt="Screen Shot 2021-02-23 at 16 19 20" src="https://user-images.githubusercontent.com/60556879/108856853-26990f80-75f3-11eb-94e9-c018c3b7b406.png">

Reply:
I think while this is not editing page we don't need to make them visible